kdevnel/ynab-absa-csv-convert-js
# Added visual improvements to how data is displayed

Closes #4 

## Description
This release brings an update to the way the data is visualised on the page.

The CSV data is now displayed in a table after it's been processed to make it easier to see in a more appealing way. This is a temporary step until the CSV file exporter is complete and reduces reliance on test code to view data.

This PR also includes some additonal on-page styling via CSS.

## Testing

- Download the codebase to your computer
- Open index.html in your preferred browser (tested with Firefox)
- Use the /testing/input-sample.csv file in the file uploader
- You should an array of objects logged to the console and a table containing the formatted CSV data on the page
